system/lib/libc/emscripten_memcpy.c

```diff
@@ -9,6 +9,13 @@
 // HEAPU8.set()
 extern void *emscripten_memcpy_big(void *restrict dest, const void *restrict src, size_t n);
 
+// XXX EMSCRIPTEN ASAN: build an uninstrumented version of memcpy
+#if defined(__EMSCRIPTEN__) && defined(__has_feature)
+#if __has_feature(address_sanitizer)
+#define memcpy __attribute__((no_sanitize("address"))) emscripten_builtin_memcpy
```

Comment:
It would be good to test this with a `-mbulk-memory` build just to make sure everything still works correctly when `memcpy` and friends are lowered to bulk memory instructions.

Reply:
This appear fine. If ASan is used, it turns the call into `__asan_memcpy`.